### PR TITLE
Make shift laser gun

### PR DESCRIPTION
**Pull request by UKNOWH**
## About the PR 
Adds the construction graph for makeshift laser
**Screenshots**

**Changelog**

:cl:
- add: Added makeshift laser gun!

### CODE REVIEW

**Comment by Veritius** [comment]
The techfab circuit board should be mentioned in the PR description and changelog.

File: Resources/Prototypes/Recipes/Construction/Graphs/weapons/makeshift laser gun.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        - to: makeshiftLaser
+          steps:
+            - material: Cable
+              amount: 2
+              doAfter: 2
+            - material: Steel
+              amount: 10
+              doAfter: 4
+            - prototype: MaterialDiamond1
+              name: refined diamond
```
Comment (Veritius) [inline Resources/Prototypes/Recipes/Construction/Graphs/weapons/makeshift laser gun.yml:16]: The refined diamond prototype should be defined with everything else, for the sake of organisation and potential re-use in later recipes.

**Review by Veritius — CHANGES_REQUESTED** [review]

**Comment by Veritius** [comment]
I made a dumbass suggestion and removed it. Apparently GitHub still keeps the little thingy. Oops.

**Comment by Laus4Deus** [comment]
👀 what's up with those files changed

**Comment by EmoGarbage404** [comment]
i believe there might be a forking issue, causing files from your previous pr to be on this one. They should be removed before this is merged.

**Comment by mirrorcult** [comment]
does this even work anymore now that laser cells arent ejectable

**Comment by mirrorcult** [comment]
i dont really think this should be in regardless design wise

**Comment by wrexbe** [comment]
Do you have a different recipe in mind, or just remove the makeshift laser?

**Comment by Peptide90** [comment]
> Do you have a different recipe in mind, or just remove the makeshift laser?

I think mirror doesn’t want mining mains mass producing laser guns. 

Imo if it stays maybe require a battery, actual laser and some metal as the graph doesn’t make a ton of sense right now. Then salvage will have to get lucky on a ship (don’t think any have lasers) or cooperate with science.

**Pull request merged**